### ISSUE TITLE
Warning: getHexString - ignoring invalid character:

### ISSUE THREAD

**Issue by gouteru** [TaTo30/vue-pdf#114]
- vue-pdf 1.10

When I tried to view an example pdf file,  nothing was displayed and many of console output below.

Warning: getHexString - ignoring invalid character: 33
and many of the error log  with different character numbers were output.

The src url of example pdf is 
'https://raw.githubusercontent.com/mozilla/pdf.js/ba2edeae/examples/learning/helloworld.pdf'

**Comment by NetMango** [comment]
Yes, me too:

`Warning: getHexString - ignoring invalid character: 33
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 79
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 84
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 89
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 80
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring additional invalid characters.
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 104
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 116
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 109
2
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 108
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring additional invalid characters.
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 104
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 115
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 114
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 105
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 112
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 116
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring additional invalid characters.
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 47
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 115
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 114
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 105
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 112
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring additional invalid characters.
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 116
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 105
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 116
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 108
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 33
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 79
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 84
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 89
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 80
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring additional invalid characters.
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 104
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 116
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 109
2
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 108
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring additional invalid characters.
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: getHexString - ignoring invalid character: 104
data:application/jav…Z2VIYW5kbGVyfTs=:21 Warning: Indexing all PDF objects`

**Comment by TaTo30** [comment]
I would need a code sample because I can't reproduce it.

![imagen](https://github.com/TaTo30/vue-pdf/assets/57086025/8a457874-1a22-4eb7-878a-c457c3b9a6be)

**Comment by atrityuser93** [comment]
This happened in my case when the pdf file was not in the public directory of my Vue application. Once I moved the file to the public directory and accordingly changed the file path when create the usePDF instance, everything turned out good.
Also, I had to copy and paste the cMaps directory into my public folder. However, moving only the cMaps folder to the public directory did help. Doing both the above steps helped in my case.
